app/src/main/java/org/glucosio/android/activity/AddReadingActivity.java

```diff
@@ -134,6 +134,7 @@ public void run() {
                 AnimationTools.startCircularReveal(doneFAB);
             }
         };
+        doneFAB.postDelayed(fabAnimationRunnable, 600);
```

Comment:
Please don't forget to remove callbacks when activity is destroyed. This will ensure no memory leaks (even for such short period 600ms).